projects/canopy/src/lib/filter-buttons/filter-button.stories.ts

```diff
@@ -0,0 +1,62 @@
+import { ReactiveFormsModule } from '@angular/forms';
+
+import { withKnobs, boolean, text } from '@storybook/addon-knobs';
+import { moduleMetadata } from '@storybook/angular';
+import { action } from '@storybook/addon-actions';
+
+import { LgFilterButtonModule } from './filter-button.module';
+import { ReactiveFormMultipleComponent } from './filter-button/filter-button-select-multiple.stories';
+import { ReactiveFormFilterSelectOneComponent } from './filter-button/filter-button-select-one.stories';
+import { notes } from './filter-button.notes';
+
+export default {
+  title: 'Components/Filter Buttons',
+  parameters: {
+    decorators: [
+      withKnobs,
+      moduleMetadata({
+        imports: [ReactiveFormsModule, LgFilterButtonModule],
+        declarations: [
+          ReactiveFormFilterSelectOneComponent,
+          ReactiveFormMultipleComponent,
+        ],
+      }),
+    ],
+    'in-dsm': {
+      id: '5f75aab1766566c38799e764',
+    },
+    notes: {
+      markdown: notes,
+    },
+  },
+};
+
+export const selectOne = () => ({
+  template: `
+        <lg-reactive-form
+        [disabled]="disabled"
+        [label]="label"
+        (datesChange)="datesChange($event)">
+    </lg-reactive-form>
+    `,
+  props: {
+    label: text('label', 'Select a date range'),
+    datesChange: action('datesChange'),
```

Comment:
```suggestion
    dateChange: action('dateChange'),
```

Reply:
I think the singular is more standard, but super minor.

Reply:
Done